masterlist.yaml

```diff
@@ -18091,6 +18108,15 @@ plugins:
     after:
       - 'Qw_ACE_CRF Patch.esp'
       - 'Qw_WACCF_CRF Patch.esp'
+          
+  - name: 'UltimateCollege.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/44797' ]
+    msg:
+      - <<: *compatNotes
+        subs: [https://www.nexusmods.com/skyrimspecialedition/articles/2761]
```

Comment:
[Installation and Compatibility](https://www.nexusmods.com/skyrimspecialedition/articles/2761)

>  Installation:
> 
> - Load ICOW BEFORE OCOW. This is very important or the whole mod will be a mess.
> - DO NOT load OCW_CellSettings.esp. All cell settings are included in this patch.

I can't recall if any patches from OCW require `OCW_CellSettings.esp`, if they do then we'd need to disable those patch messages.

> - This mod should be towards the end of your load order with other compatibility patches as it is a patch at its core.

`UltimateCollege.esp` would need to have the same sorting rules as `OCW_CellSettings.esp`.

```yaml
  - name: 'OCW_CellSettings.esp'
    group: *cellSettingsGroup
    after:
      - 'ELE_SSE.esp'
      - 'ELFXEnhancer.esp'
      - 'ELFX - Hardcore.esp'
      - 'Immersive Music.esp'
      - 'Immersive Music Replacer.esp'
      - 'Luminosity Lighting Overhaul.esp'
      - 'Luminosity - Skyrim is Lit Edition.esp'
      - 'Soulmancer Soundtrack - NoVindDun.esp'
      - 'Unique Region Names.esp'
      # Group afters, to prevent sorting errors
      - 'MissingEZs_AllExteriors.esp'
      - 'MissingEZsFixed.esp'
      - 'NoGrassINCaves.esp'
````